unfoldingWord/translationCore
# Completed Tool Main Page (HomeScreen)

### This pull request addresses:

- Created ToolCard Component.
- Now when selecting a project it automatically takes you to the next step (Tools main page).
- Now when you click the Home button in the Status bar it takes you to the HomeScreen.

### How to test this pull request:

- Pull `feature/manny/tool-main-page/976` branch from both `tN` and `tW`.
- Select a project. 
    (The toolCards page should automatically open and all tool cards should have accurate progress in the progress bar)
- Try loading a project with both tools using the tool cards in the Tool Main Page.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2119)
